examples/file/stream/README.md

```diff
@@ -0,0 +1,32 @@
+File Stream Example
+=================
+
+This is an example console application demonstrating how to use the
+SystemLink File API to upload files from an in-memory stream.
```

Comment:
> from an in-memory stream

Might be a silly question, but do we need to specify whether this is on a server or client?